dts/bindings/display/frida,nt35510.yaml

```diff
@@ -1,7 +1,11 @@
 # Copyright (c) 2024 Erik Andersson <erian747@gmail.com>
 # SPDX-License-Identifier: Apache-2.0
 
-description: Frida Techshine Panel
+description: |
+    Frida Techshine Panel
```

Comment:
you might as well move all of these to be the `title:`, since you're here and cleaning this up nicely.